$MirOS: ports/devel/cvs/patches/patch-configure_in,v 1.2 2010/09/15 20:56:57 tg Exp $
--- configure.in.orig	Mon Oct  3 13:34:23 2005
+++ configure.in	Thu Sep 16 00:11:41 2010
@@ -1,3 +1,5 @@
+dnl $MirOS: ports/devel/cvs/patches/patch-configure_in,v 1.2 2010/09/15 20:56:57 tg Exp $
+dnl
 dnl configure.in for cvs
 
 AC_COPYRIGHT(
@@ -15,8 +17,8 @@ but WITHOUT ANY WARRANTY; without even t
 MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 GNU General Public License for more details.])
 
-AC_INIT([Concurrent Versions System (CVS)],[1.12.13],
-	[bug-cvs@nongnu.org],[cvs])
+AC_INIT([Concurrent Versions System (CVS)],[1.12.13-MirPorts-QQ],
+	[miros-discuss@mirbsd.org],[cvs])
 AC_CONFIG_SRCDIR(src/cvs.h)
 AC_CONFIG_AUX_DIR(build-aux)
 AM_INIT_AUTOMAKE([gnu 1.9.2 dist-bzip2 no-define])
@@ -110,7 +112,7 @@ dnl
 dnl Yeah, put the value in a variable add it to the substitution list
 dnl then have configure create sanity.sh from sanity.sh.in...
 glocs="$PATH:/usr/local/bin:/usr/contrib/bin:/usr/gnu/bin:/local/bin:/local/gnu/bin:/gnu/bin"
-AC_PATH_PROGS(ROFF, groff roff, $missing_dir/missing roff, $glocs)
+AC_PATH_PROGS(ROFF, nroff gnroff groff roff, $missing_dir/missing roff, $glocs)
 AC_PATH_PROG(PS2PDF, ps2pdf, $missing_dir/missing ps2pdf)
 AC_PATH_PROG(TEXI2DVI, texi2dvi, $missing_dir/missing texi2dvi)
 
@@ -180,6 +182,7 @@ AC_CHECK_MEMBERS([struct stat.st_blksize
 AC_CHECK_MEMBERS([struct stat.st_rdev])
 
 AC_FUNC_FSEEKO
+AC_CHECK_FUNC(fseeko)
 if test $ac_cv_func_fseeko = no; then
     AC_LIBOBJ(fseeko)
     AC_LIBOBJ(ftello)
@@ -1444,7 +1447,7 @@ AC_ARG_ENABLE(
   [config-override],
   AC_HELP_STRING(
     [--enable-config-override],
-    [Set to a comma-seperated list of paths to directories (designated by
+    [Set to a comma-separated list of paths to directories (designated by
      trailing `/') and files, specifies the path prefixes (for directories) and
      paths to files the CVS server commands will allow configuration to be read
      from.  Specify `--enable-config-override=no' to disable config file
